pages/posts/why-portal-needs-to-be-its-own-network.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Why Portal Needs To Be Its Own Network
+date: 2024/03/12
+description: Explaining why the Portal Network needed to build its own network rather than leverage an existing distributed storage engine like Filecoin or Swarm.
+tag: project, Ethereum, p2p, storage
+author: Piper Merriam
+---
+
+We commonly get asked why we don't use one of the various general purpose
+distributed storage networks (like Filecoin/IPFS).  Baked into this question is
```

Comment:
Might be worth mention `bittorrent` up here, since that's the one I generally hear the most questions about.

Reply:
I agree that bittorrent would be a good example to mention.
Especially as it can be used to explain how its usage would differ from Portal history network. That is, bittorrent could be a complementary solution to Portal network for EIP-4444.
Where Portal shines in individual quick requests, bittorrent could be an alternative for large bulk download, at least for history data (e.g. era1 files over bittorrent).